packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -490,6 +490,59 @@ test.describe('PieInput - Component tests', () => {
                 expect(await page.evaluate(() => document.querySelector('pie-input')?.value)).toBe('foo');
             });
         });
+
+        test.describe('disabled', () => {
+            test('should not disable the component if the prop is not provided', async ({ mount }) => {
+                // Arrange
+                const component = await mount(PieInput, {});
+
+                // Act
+                const input = await component.locator('input');
```

Comment:
I'm not sure you need the awaits here. You may need it on the expect lines but for the locator itself i dont think it returns a promise.

Reply:
Good shout. There is indeed a (very subtle) IDE warning about this which I completely missed. I'll resolve all instances of that in this file.

Reply:
What's strange is the Playwright Docs all use await in their locator examples, which is probably how it found it's way into the code base

https://playwright.dev/docs/locators